asv_bench/benchmarks/algorithms.py

```diff
@@ -5,6 +5,7 @@
 from pandas._libs import lib
 
 import pandas as pd
+from pandas.core.arrays.string_arrow import ArrowStringDtype
```

Comment:
Can you do this in a try/except? (we need to be able to still run the benchmarks with slightly older pandas version that might not have this import available)